connect.go

```diff
@@ -152,7 +152,7 @@ func Connect(connectStr string) (success bool) {
 
 	// Wait for a clear channel
 	switch url.Scheme {
-	case MethodArdop:
+	case MethodArdop, MethodVaraFM, MethodVaraHF:
 		waitBusy(adTNC)
```

Comment:
This must also be changed. Passing adTNC here is only correct for ARDOP. We will need to pass a reference to the appropriate TNC connection, which depend on the URL scheme.